beakerx/gradlew

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
If we leave original gradlew, then we have to duplicate folders with gradle wrappers because script point to current project directory. Is that ok?